Weather Alert.xcodeproj/project.pbxproj.orig

```diff
@@ -0,0 +1,521 @@
+// !$*UTF8*$!
```

Comment:
Can you please delete this `.orig` file (`/project.pbxproj.orig`). `.orig` gets created for you when you do a merge - we don't need them anymore if merge has been successful. Please delete any other `.orig` files as well.